en_US/data-integration/data-bridge-clickhouse.md

```diff
@@ -0,0 +1,151 @@
+- Basic knowledge of UNIX terminal and commands 
+
+:::
+
+## Feature List
+
+- [Connection pool](./data-bridges.md)
+- [Async mode](./data-bridges.md)
+- [Batch mode](./data-bridges.md)
+- [Buffer mode](./data-bridges.md)
+
+## Quick Start Tutorial
+
+This section introduces how to use the ClickHouse bridge with a practical tutorial, covering topics like how to create a Clickouse server, how to set up a bridge, and how to set up a rule for forwarding data to the bridge and testing that it all works. 
+
+This tutorial assumes that you run both EMQX and ClickHouse on the local machine. If you have ClickHouse and EMQX running remotely, please adjust the settings accordingly.
+
+### Start a ClickHouse Server
+
+This section introduces how to start a ClickHouse server using [Docker](https://www.docker.com/). 
+
+1. Create a file called `init.sql` with the following initialization SQL statements, which will help to initialize the database when the container starts up.
+
+   ```bash
+   cat >init.sql <<SQL_INIT
+   CREATE DATABASE IF NOT EXISTS mqtt_data;
+   CREATE TABLE IF NOT EXISTS mqtt_data.messages (
+       data String,
+       arrived BIGINT
+   ) ENGINE = Memory;
+   SQL_INIT
+   ```
+
+2. Then, start a ClickHouse server using the following command, which defines the database name, port number, user name and password. And it will also mount the `init.sql` file in the current directory to the docker directory.
+
+   ```bash
+   docker run \
+   --rm \
+   -e CLICKHOUSE_DB=mqtt_data \
+   -e CLICKHOUSE_USER=emqx \
+   -e CLICKHOUSE_DEFAULT_ACCESS_MANAGEMENT=1 \
+   -e CLICKHOUSE_PASSWORD=public \
+   -p 18123:8123 \
+   -p19000:9000 \
+   --ulimit nofile=262144:262144 \
+   -v ./init.sql:/docker-entrypoint-initdb.d/init.sql \
+   clickhouse/clickhouse-server
+   ```
+
+You can find more information about running ClickHouse in docker [on dockerhub](https://hub.docker.com/r/clickhouse/clickhouse-server).
+
+### Create a ClickHouse Data Bridge
+
+Then you can start to create an EMQX data bridge to ClickHouse.
+
+1. Go to EMQX Dashboard, click **Data Integration** -> **Data Bridge**.
+
+2. Click **Create** on the top right corner of the page.
+
+3. In the **Create Data Bridge** page, click to select **ClickHouse**, and then click **Next**.
+
+4. Input a name for the data bridge. The name should be a combination of upper/lower case letters and numbers.
+
+5. Input the connection information:
+   *  **Server URL**: Input **http://127.0.0.1:18123**, or the actual URL if the ClickHouse server is running remotely.
+   * **Database Name**: Input **mqtt_data**.
+   * **Username**: Input **emqx**.
+   * **Password**: Input **public**
+   
+7. **Batch Separator** (optional): for separate multiple items, default:  `,`  . You can customize the separate based on the data input/out format you defined in ClickHouse, for example, if [JSONCompactEachRow` format](https://clickhouse.com/docs/en/interfaces/formats#jsoncompacteachrow)  is used, should leave it blank. 
```

Comment:
I don't think this is easy to understand for anyone that does not already know what it is. How would the user now at this point what it means to separate multiple items. How would the user even know what items refer to? I think some context has been lost from the original description (for example, that this is something that is only relevant to batching).

> You can customize the separate based on the data input/out format you defined in ClickHouse

I don't think this is a good way of describing it. The format is not specified inside clickhouse. It is specified in the SQL statement. I think the original text had a reference to the clickhouse syntax for INSERT statements. 

the separate -> the separator

My suggestion is to put this description as a separate section as it originally was and try to make it easier to understand. Perhaps it would be useful with a an example showing an SQL statement that use a different separator.

Perhaps you can keep this item but rewrite it to something like this:

In this example you can keep the default value "`,`" for this setting. This setting only needs to be changed if you enable [batching](link to batching desc) for the bridge and if you specify an alternative format with [ClickHouse's FORMAT syntax](link to clickhouse's doc for the INSERT statement). For further info see [advanced settings](link to advanced settings).